js/visualization_entity_tables_edit.js

```diff
@@ -1,13 +1,25 @@
 (function($) {
+
   $(document).on('ready', function(){
+
+    var settings = JSON.parse($('#edit-field-ve-settings-und-0-value').val());
+    console.log(settings);
+
     var $resourceField = $('#edit-field-uuid-resource-und-0-target-uuid');
+    var $resizeField = $('#edit-field-ve-table-resize-und');
+    var $settingsField = $('#edit-field-ve-settings-und-0-value');
 
     $resourceField.on('autocompleteSelect', function(event, node) {
       var re = /\[(.*?)\]/;
       var uuid = re.exec($resourceField.val())[1];
-      $('#edit-field-ve-settings-und-0-value').val(JSON.stringify({source:'/node/' + uuid + '/download'}));
+      settings.source = '/oklahoma/node/' + uuid + '/download';
```

Comment:
We need to remove this before merge to prevent this library remain tied to the OK project.